packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -1502,6 +1518,14 @@ addFieldsDict(Users, {
     hidden: true,
     canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
   },
+  usernameUnset: {
+    type: Boolean,
+    optional: true,
+    canRead: ['members'],
+    hidden: true,
+    canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
```

Comment:
I think `userOwns` shouldn't be in here.The `NewUserCompleteProfile` mutation doesn't need it (since it's a dedicated mutation not going through the CRUD stuff), and there's potentially mild mischief that one can achieve by resetting this to true and changing your username repeatedly.

Reply:
Thanks, good catch.